plasmapy/atomic/_exceptions.py

```diff
@@ -0,0 +1,57 @@
+class AtomicError(Exception):
```

Comment:
My preference would be to name the file `exceptions.py`.  I'm thinking that this shouldn't be hidden because users may need to import these exceptions in order to do exception handling of `atomic` errors in their own programs.

Reply:
Done!